pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -85,6 +85,8 @@ type LoadBalancerOpts struct {
 	MonitorMaxRetries    uint       `gcfg:"monitor-max-retries"`
 	ManageSecurityGroups bool       `gcfg:"manage-security-groups"`
 	NodeSecurityGroupID  string     `gcfg:"node-security-group"`
+
+	EnableAnnotationFloatingNetworkId bool `gcfg:"enable-annotation-floating-network-id"` // allow override of floatingnetworkid in service
```

Comment:
in that another PR there were some discussion that admins should have possibility to disable this feature, because then users have "too much power". But well, I agree with you also that we do not need it.